Felienne/hedy
# [CLEAN-UP] Remove redundant duplicate filter when verifying achievements

**Description**
The previous bug of duplicate achievement storing no longer occurs due to our re-writing into a session-based tracking of achievement. The cause is still unknown, but the problem is fixed. Therefore the `duplicate_filter` on the `add_achievement_to_username` can be removed as it is redundant. Therefore, this PR doesn't actually solve the issue but removes the bucket-fix created earlier to temporary solve the issue.

**Fix for**
This PR fixes #1601

**How to test**
As this is a clean-up task no difference should be noticed. One can verify the correctness by emptying their local `Achievements` table and notice that no duplicates are stored when receiving different achievements.

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
